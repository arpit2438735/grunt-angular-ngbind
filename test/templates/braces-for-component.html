<div>
    <angular-component>{{::testBraces}}</angular-component>
    <angular-second>{{testBraces}}</angular-second>
    <i class="some-style">{{::testBraces}}</i>
</div>
