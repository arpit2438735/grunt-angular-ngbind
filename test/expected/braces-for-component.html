<div>
    <angular-component ng-bind="::testBraces"></angular-component>
    <angular-second ng-bind="testBraces"></angular-second>
    <i class="some-style" ng-bind="::testBraces"></i>
</div>
